http/http-advanced/keycloak-realm.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Why do we need this real on the root module folder?

Reply:
We do not. Removed it.